client/Components/ProductInfo.jsx

```diff
@@ -8,20 +8,31 @@ export default class ProductInfo extends React.Component {
     super(props);
     this.state = {
       productId: '',
+      searchTerm: '',
       value: '',
       productInfo: {},
     };
-    // const myScript = document.getElementById('bundle');
   }
 
   componentDidMount() {
-    this.setState({
-      productId: [window.location.href.split('/')[4]],
-    }, () => {
-      fetch(`http://localhost:3002/products/${this.state.productId}/description-and-standards`).then((res) => res.json()).then((data) => this.setState({
-        productInfo: data,
-      })).catch((err) => console.log(err));
-    });
+    const param = [window.location.href.split('/')[4]];
```

Comment:
Is search term and product Id always the same value?

Reply:
No, search term would be a string and product id would be the unique identifier of a product. This is set up to allow for both, but the front end doesn't accommodate both (yet)